src/BenchmarkDotNet.Core/Helpers/ResourceHelper.cs

```diff
@@ -4,11 +4,24 @@
 
 namespace BenchmarkDotNet.Helpers
 {
-    internal static class ResourceHelper
+    internal class ResourceHelper
     {
-        internal static string LoadTemplate(string name)
+        private readonly string prefix1;
```

Comment:
the names prefix1 and prefix2 don't tell anything to the user.
I looked at the code and it seems that there is no need for having two prefixes.

Suggestion: 

- please rename prefix1 to namespacePrefix
- please remove prefix2
- please add a `/// <summary>` comment: the namespace of searched resource file is required to read it from dll